docs: `*_invitation` life cycle clarification (#756)
mongodb/terraform-provider-mongodbatlas: website/docs/r/org_invitation.html.markdown

* ORG_READ_ONLY
* ORG_MEMBER

> ~> **IMPORTANT:**
> This resource is specifically to mange the invitations, not the user beyond that. Possible provider behavior depending on the invitee's action:

Reviewer: Just a slight change that should make it even clearer:

```suggestion
~> **IMPORTANT:** This resource is only for managing invitations, not for managing the Atlas User being invited. Possible provider behavior depending on the invitee's action:
```
